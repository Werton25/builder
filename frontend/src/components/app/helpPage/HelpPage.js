import React from 'react';
import HelpContent from "./HelpContent";

export default class HelpPage extends React.Component {
  render() {
    return (
      <div className='container'>
        <h4>Горячие клавиши редактора запросов</h4>
        <HelpContent/>
      </div>
    )
  }
}
