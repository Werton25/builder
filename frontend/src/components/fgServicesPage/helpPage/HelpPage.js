import {div, h4, table, tbody, td, th, thead, tr, p} from "../../elements";
import React from 'react';

export default class FaqPage extends React.Component {
  render() {
    return div({className: 'container'},
      h4({}, 'Горячие клавиши редактора запросов'),
      table({className: 'table'},
        thead({},
          tr({},
            th({}, 'Сочетание клавиш'),
            th({}, 'Описание')
          )
        ),
        tbody({},
          tr({},
            td({}, 'Ctrl + Enter'),
            td({}, 'Включить полноэкранный режим'),
          ),
          tr({},
            td({}, 'Ctrl + F'),
            td({}, 'Поиск по тексту'),
          ),
          tr({},
            td({}, 'Ctrl + Alt + L'),
            td({}, 'Отформатировать текст'),
          ),
        )
      )
    )
  }
}
