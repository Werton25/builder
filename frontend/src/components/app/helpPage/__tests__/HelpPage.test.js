import React from "react";
import renderer from "react-test-renderer";
import HelpPage from "../HelpPage";

describe('HelpPage tests', () => {
  it('HelpPage renders correctly', () => {
    const tree = renderer
      .create(
          <HelpPage/>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});