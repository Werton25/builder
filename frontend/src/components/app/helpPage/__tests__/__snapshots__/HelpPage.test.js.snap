// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpPage tests HelpPage renders correctly 1`] = `
<div
  className="container"
>
  <h4>
    Горячие клавиши редактора запросов
  </h4>
</div>
`;
